.gitignore

```diff
@@ -1 +1,2 @@
 Gemfile.lock
+spec/dummy/log
```

Comment:
I don't think this is working. I still see the log files in the PR

Reply:
They should be gone now